migrationmanagers/migrationmanagers.go

```diff
@@ -69,17 +69,17 @@ func (m *MigrationManager) Save() error {
 	return schema.Write(m.schema)
 }
 
-// Run applies a migration to the TestTrack server, writing out the schema
-func (m *MigrationManager) Run(migrationRepo migrations.Repository) error {
-	err := m.Apply(migrationRepo)
+// Migrate syncs a migration and its version to the TestTrack server
+func (m *MigrationManager) Migrate() error {
+	err := m.Sync()
```

Comment:
`Run` was basically replaced with a slightly different method, `Migrate`. We used to write out the schema to disk a lot, even when applying migrations that were already reflected in the schema. This isn't really necessary, so we stopped doing it. If we want to verify whether the schema would differ as a result of applying outstanding migrations, we can always add the check command (which I have on another branch) rather than just end up with a modified schema file on the filesystem after the fact.